svg/elements/svg.json

```diff
@@ -56,9 +56,7 @@
                 "version_added": "1"
               },
               "chrome_android": "mirror",
-              "edge": {
-                "version_added": "≤79"
-              },
+              "edge": "mirror",
```

Comment:
Features like these that have been supported since the early beginnings of Chrome and Firefox, were likely also available in Edge.